tests/platform/mellanox/minimum_table.py

```diff
@@ -0,0 +1,90 @@
+MINIMUM_TABLE= {
+    'ACS-MSN2700': {
+        "p2c_trust":   {"-127:40":13, "41:120":15},
+        "p2c_untrust": {"-127:25":13, "26:30":14 , "31:35":15, "36:120":16},
+        "c2p_trust":   {"-127:20":13, "21:25":14 , "26:30":15, "31:120":16},
+        "c2p_untrust": {"-127:20":13, "21:25":14 , "26:30":15, "31:120":16},
+        "unk_trust":   {"-127:20":13, "21:25":14 , "26:30":15, "31:120":16},
+        "unk_untrust": {"-127:20":13, "21:25":14 , "26:30":15, "31:120":16}
+    },
+    'LS-SN2700': {
```

Comment:
Where does this LS-SN2700 come from?

Reply:
It's one SKU of 2700, I just copy the SKU name from existing code.